website/static/js/fangorn.js

```diff
@@ -642,7 +642,7 @@ function _fangornActionColumn (item, col) {
 
     // Upload button if this is a folder
     // If File and FileRead are not defined dropzone is not supported and neither is uploads
-    if (window.File && window.FileReader && item.kind === 'folder' && item.data.provider && item.data.permissions.edit) {
+    if (window.File && window.FileReader && item.kind === 'folder' && item.data.provider && item.data.permissions.edit && !item.data.permissions.diskSavingMode) {
```

Comment:
Do you need to add `diskSavingMode` to every item? Can't you just check the value of `tb.options.uploads`? Is `tb.options.uploads` false when DISK_SAVING_MODE is enabled?